cmd/ocm-backplane/login/login.go

```diff
@@ -352,3 +372,26 @@ func doLogin(api, clusterID, accessToken string) (string, error) {
 
 	return api + *loginResp.JSON200.ProxyUri, nil
 }
+
+func listNamespaces(clusterID, clusterName, env string, isHypershift bool) []string {
+
+	klusterletPrefix := "klusterlet-"
+	hivePrefix := fmt.Sprintf("uhc-%s-", env)
+	hcpPrefix := fmt.Sprintf("ocm-%s-", env)
+
+	var nsList []string
+
+	if isHypershift {
+		nsList = []string{
+			klusterletPrefix + clusterID,
+			hcpPrefix + clusterID,
+			hcpPrefix + clusterID + "-" + clusterName,
+		}
+	} else {
+		nsList = []string{
+			hivePrefix + clusterID,
+		}
+	}
+
+	return nsList
+}
```

Comment:
What if we get the matching namespaces via bp-api endpoint ?  basically `oc get ns -A| grep {cluster-internal-id}`

Reply:
The namespace pattern is fixed so it should be ok to print them directly.

Reply:
Is this pattern unique to the FedRamp stack too?